flutter-gui-tests/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,6 @@
+#Fri Jun 23 08:50:38 CEST 2017
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+zipStoreBase=GRADLE_USER_HOME
+zipStorePath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.2-all.zip
```

Comment:
I notice the timestamp is a bit old.  Any reason we're not pointing to a more recent gradle distro?

Looking on http://services.gradle.org/distributions/, I see they're up to `5.*`.

(Or maybe this just doesn't matter.)

Reply:
I chose 4.10.2 since it was the latest I had prior to 5.0, and 5.0 introduces some breaking changes.

Reply:
😬 

Maybe update with a comment to that effect?